openquakeplatform/templates/includes/sidebar.html

```diff
@@ -0,0 +1,12 @@
+<div class="oq-button-home">
+      <div class="col-md-4">
+        <p><a class="oq-button-home-btn btn" href="/account/login/?next=/maps/" role="button">Maps</a></p>
```

Comment:
Please use URL resolver instead of static urls, something like `{% url "account_login" %}?next={{next}}`